e2e_tests/tests/cluster/test_checkpoints.py

```diff
@@ -207,64 +198,43 @@ def run_gc_checkpoints_test(checkpoint_storage: Dict[str, str]) -> None:
     sess = api_utils.user_session()
     fixtures = [
         (
-            conf.fixtures_path("no_op/gc_checkpoints_decreasing.yaml"),
+            noop.traininglike_steps(10, scale=0.9),
             {
-                (bindings.experimentv1State.COMPLETED.value): {800, 900, 1000},
-                (bindings.experimentv1State.DELETED.value): {
-                    100,
-                    200,
-                    300,
-                    400,
-                    500,
-                    600,
-                    700,
-                },
+                (bindings.experimentv1State.COMPLETED.value): {8, 9, 10},
```

Comment:
why are some of these using bindings.state and others are using `client.Experiment.state`

Reply:
uh, because `exp_ref.wait()` returns a client.ExpeirmentState, and `exp.wait_for_experiment_state()` returns a bindings.state.

And because I switched from preferring exp.wait_for_experiment_state to exp_ref.wait() halfway through :disappointed: 

Will fix.

Reply:
fixed.